tests/e2e/README.md

```diff
@@ -0,0 +1,66 @@
+# End-to-end (e2e) Tests for the wp-parsely Plugin
+
+This suite is meant to simulate actual user actions as they interact with this plugin. Tests are run against a "real" WordPress instance and activities are performed in a "real" browser. The idea is that we can provide confidence that changes going forward have the intended effect on the DOM and rendered content that plugin users and site visitors will see under various specific conditions.
+
+## How it Works
+
+In order for the tests to do their job, they need a back end that simulates a WordPress instance. To that end, we spin up a bare-bones containerized site and configure it to the default values for the WordPress `e2e-tests` helper.
+
+Once there's a functioning back end, we leverage the `@wordpress/scripts` utility's [built-in functionality](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-scripts/#test-e2e) to launch a browser via [puppeteer](https://pptr.dev/).
+
+The tests use the [Jest framework](https://jestjs.io/) to drive a user flow and assert on expected outcomes. In addition to the [Puppeteer API](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md), there are a number of helpers to accomplish frequently performed tasks in the [`@wordpress/e2e-test-utils` package](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-e2e-test-utils/).
+
+See this post for more infomation: https://make.wordpress.org/core/2019/06/27/introducing-the-wordpress-e2e-tests/
+
+## How to Run
+
+- Provision the back end
+
+  - Make sure [`docker-compose` is installed](https://docs.docker.com/compose/install/).
+  - From the top-level directory, run:
+
+    `docker-compose -f tests/e2e/docker-compose.yml up`
+
+    Tip: if you cd into this directory, you can simply run `docker-compose` & omit the `-f tests/e2e/docker-compose.yml` from these commands
+
+  - Once you see a line that says:
+
+    `Success: WordPress installed successfully.`
```

Comment:
Is this a different message to the "WordPress is ready!" message in the GitHub workflow?